sky/utils/schemas.py

```diff
@@ -328,10 +328,7 @@ def get_service_schema():
                     'max_replicas': {
                         'type': 'integer',
                     },
-                    'qps_upper_threshold': {
-                        'type': 'number',
-                    },
-                    'qps_lower_threshold': {
```

Comment:
For backward compatibility, we should still enable these in schema check and throw an error with more information in `__init__` of `SkyServiceSpec`. We could have a todo here say remove this after 2 minor release ;)